requirements.txt

```diff
@@ -3,6 +3,7 @@ boto3==1.4.7
 coverage==4.4.1
 enum34==1.1.6
 flake8>=2.5.1,<2.6
+simplejson>=3.12.2
```

Comment:
do you import this library somewhere in the python implementation? because I don’t see that you are using it.

Reply:
right, not anymore, will clean up. basically it should be codacy reporting such issues, not a human